clients/include/rocblas.hpp

```diff
@@ -145,6 +145,51 @@ static constexpr auto rocblas_asum<rocblas_float_complex, float> = rocblas_scasu
 template <>
 static constexpr auto rocblas_asum<rocblas_double_complex, double> = rocblas_dzasum;
 
+// asum_batched
+template <typename T1, typename T2>
+rocblas_status (*rocblas_asum_batched)(rocblas_handle  handle,
+                                       rocblas_int     n,
+                                       const T1* const x[],
+                                       rocblas_int     incx,
+                                       T2*             result,
```

Comment:
Reminder from an earlier email

"Where Legacy BLAS functions have return values, the return value is instead added as the last function argument. ... This applies to the following functions: xDOT, xDOTU, xNRM2, xASUM, IxAMAX, IxAMIN.

For the batch and strided batched functions, note that the last argument should be the return, not batch_count. "